pkg/test/usersignup/usersignup.go

```diff
@@ -137,6 +139,10 @@ func WithActivations(value string) Modifier {
 	return WithAnnotation(toolchainv1alpha1.UserSignupActivationCounterAnnotationKey, value)
 }
 
+func WithVerificationAttempts(value int) Modifier {
+	return WithAnnotation(toolchainv1alpha1.UserVerificationAttemptsAnnotationKey, strconv.Itoa(value))
```

Comment:
Is `UserVerificationAttemptsAnnotationKey` clear enough or could it be confused between activation code and phone code verification?

Reply:
I believe we could use the same key, with a max attempts based on any combination of attempts with a phone code or an activation code. I mean, I believe it's sufficient to give users a max number of attempts, whatever mechanism they choose to use. And besides, it does not impact the logic on the back-end ;)

I hope this makes sense.

Reply:
I think that's fine since I guess it should be obvious that this UserSignup used an activation code instead of a phone verification code because it would have the activation label